<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesView</class>
 <widget class="QWidget" name="PropertiesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>953</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Property Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollAreaProperties">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>330</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>320</width>
       <height>320</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>318</width>
        <height>933</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="propertiesObject" native="true">
       <property name="geometry">
        <rect>
         <x>-8</x>
         <y>-7</y>
         <width>320</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QPushButton" name="pushButtonAttachComponent">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string> Component</string>
        </property>
        <property name="icon">
         <iconset resource="../rainbow-editor.qrc">
          <normaloff>:/res/images/attach.png</normaloff>:/res/images/attach.png</iconset>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxEnabled">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>13</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditName">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="propertySections" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>301</width>
         <height>911</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>9</y>
          <width>301</width>
          <height>891</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="PropertiesTransform" name="propertiesTransform" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>160</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="propertyEditors" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777211</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAttachCamera">
   <property name="icon">
    <iconset resource="../rainbow-editor.qrc">
     <normaloff>:/res/images/scene-camera.png</normaloff>:/res/images/scene-camera.png</iconset>
   </property>
   <property name="text">
    <string>Attach Camera</string>
   </property>
  </action>
  <action name="actionAttachLight">
   <property name="icon">
    <iconset resource="../rainbow-editor.qrc">
     <normaloff>:/res/images/scene-light-point.png</normaloff>:/res/images/scene-light-point.png</iconset>
   </property>
   <property name="text">
    <string>Attach Light</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PropertiesTransform</class>
   <extends>QWidget</extends>
   <header>PropertiesTransform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rainbow-editor.qrc"/>
 </resources>
 <connections/>
</ui>
